codes/classical/q-ary_digits/regenerating_code.yml

```diff
@@ -0,0 +1,30 @@
+#######################################################
```

Comment:
I've created a stub entry in the folder classical/properties/distributed_storage. Keep the parents/cousins as in the stub, and add the rest next committ.